app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryDescription.js

```diff
@@ -35,22 +37,30 @@ class ExpandedStoryDescription extends React.Component {
     );
   };
 
-  descriptionTextArea(description) {
-    const { onEdit, disabled } = this.props;
+  descriptionTextArea(description, suggestedUsers) {
+    const { disabled, onEdit } = this.props;
 
     return (
-      <textarea
+      <MentionsInput
         className="form-control input-sm edit-description-text"
         onChange={(event) => onEdit(event.target.value)}
         readOnly={disabled}
         value={description}
-      />
+      >
+        <Mention
+          markup="@__display__"
+          displayTransform={(id, display) => `@${display}`}
+          data={suggestedUsers}
+        />
+      </MentionsInput>
     );
   };
 
   render() {
     const { story, disabled } = this.props;
 
+    const suggestedUsers = (this.props.users && this.props.users.map(user => ({ id: user.id, display: user.username })));
```

Comment:
I think you could destructing `users` from `this.props` and create a method to map the users.
```js
const getSuggestedUsers = (users = []) => map(({ id, username }) => ({ id, display: username }));

render() {
  const { story, disabled, users } = this.props;
  const suggestedUsers = getSuggestedUsers(users);
  ...
```
🤔

Reply:
Yeah, I should 100% get users from `this.props`.